src/Mod/Mesh/BuildRegularGeoms.py

```diff
@@ -1,10 +1,11 @@
+# Copyright (c) 2005 Berthold Grupp

```

Comment:
This file starts with a Python docstring already, that is, triple quotes. This is probably the best way to do it. The author and license should be provided after this documentation block.
```py
"""Python Module for building solid regular geometric objects.
...
"""
# Copyright (c) 2005 Berthold Grupp
# License: LGPL
```

Also, the full license block should be placed explicitly like in other source files.

Reply:
Will fix this in a future commit